src/components/non-bitgo.js

```diff
@@ -4,6 +4,7 @@ import { InputField, InputTextarea, CoinDropdown, FieldTooltip } from './form-co
 import { Form, Button, Row, Col, FormGroup, Label } from 'reactstrap';
 import classNames from 'classnames';
 import ErrorMessage from './error-message';
+import * as BitGoJS from 'bitgo/dist/browser/BitGoJS.min';
```

Comment:
what's with this? we can't import the default export?

Reply:
No idea why. I'm just doing the same we did here: https://github.com/BitGo/wallet-recovery-wizard/blob/59836f1d52a5955b1efbc6cb6205611a927b839d/src/views/login/index.js#L6

Reply:
we should change this to 

`import * as BitGoJS from 'bitgo'`

that should work

Reply:
apparently this doesn't work womp womp - it'll get fixed later